docs/source/actions.rst

```diff
@@ -70,6 +70,13 @@ AWS Secret Key.
 
 Default: ``None``
 
+SPIDERMON_AWS_REGION
```

Comment:
Scrapy uses [AWS_REGION_NAME](https://docs.scrapy.org/en/latest/topics/settings.html#aws-region-name) as setting name. I would prefer if we try to keep a similar name. `SPIDERMON_AWS_REGION_NAME` is a better name.

Reply:
Absolutely! Done the change here.